js/framework/application.js

```diff
@@ -406,8 +406,8 @@ var Application = new Knowledge.Class({
         // Fixed in org.gnome.Platform//3.30. In that version, just pass strings
         // to SpawnSync() instead of byte arrays.
         function zeroTerminatedByteArray(string) {
```

Comment:
If you're updating to 3.32 then you should be able to delete this function entirely.